pygfx/materials/_points.py

```diff
@@ -244,11 +244,20 @@ class PointsMarkerMaterial(PointsMaterial):
         edge_width="f4",
     )
 
-    def __init__(self, *, marker="circle", edge_width=1, edge_color="black", **kwargs):
+    def __init__(
+        self,
+        *,
+        marker="circle",
+        edge_width=1,
+        edge_color="black",
+        custom_sdf="return -size;",
```

Comment:
did you accidentally commit this?

Reply:
It also took me a while before I figured out this is a piece of wgsl. Maybe better to default to None to avoid confusion, and in the shader do `self["custom_sdf"] = material.custom_sdf or "return -size;"`

Reply:
I definitely snuck this in late in the review. I agree with almars suggestion 

I just wanted to make something "basic" to give a display. 

The scariest thing on the GPU is when you see nothing.

Reply:
Maybe 


min(abs(coord))

Reply:
Min(abs(coord)) - size